Gemfile

```diff
@@ -27,6 +27,8 @@ gem 'bootstrap', '~> 4.3.1'
 # Bootstrap Stylesheets
 gem 'twitter-bootstrap-rails'
 
+# gem 'select2-rails', '~>3'
```

Comment:
What did you decide to use in place of this?

Reply:
Ah, this was from a previous attempt of the dropdowns. I can remove this from the Gemfile.